pkg/echopop/echopop.go

```diff
@@ -65,6 +67,66 @@ func AuthEnrollment(ic *invopop.Client) echo.MiddlewareFunc {
 
 }
 
+// AuthToken defines a middleware function that will check if the
+// header contains an authentication token.
+//
+// If it does, the token will be included in the invopop client to be used
+// to authenticate requests to the API. It is thought for endpoints where an
+// oauth access token is required to access the API.
+func AuthToken(ic *invopop.Client) echo.MiddlewareFunc {
```

Comment:
🤔  I think my impression here is that these are too use-case specific to be included in the shared library. The `X-Enrollment-ID` also feels very implementation specific, and not likely to be widely used.

Passing the Authorization headers also feels risky... you'd probably want a different name that more clearly indicates what's happening, like `ForwardAuthToken` or similar.

I recommend removing these from the client and keeping them in the Stripe app for the time being. If we find multiple uses, then it'd be a good moment to move them to the client.go package.